test/ruby/test_compile_prism.rb

```diff
@@ -962,7 +964,6 @@ def compare_eval(source)
     def assert_prism_eval(source)
       $VERBOSE, verbose_bak = nil, $VERBOSE
 
-      source = "class Prism::TestCompilePrism\n#{source}\nend"
```

Comment:
Did you mean to do this?

Reply:
Yes, otherwise we could be missing other bugs by virtue of always testing something within a class, and never isolated.

Reply:
But it looks like it's causing tests to fail, so will revert for now!